netlink-packet-sock-diag/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+authors = ["Flier Lu <flier.lu@gmail.com>", "Corentin Henry <corentinhenry@gmail.com>"]
```

Comment:
@george-hopkins I'd like to release 0.1, and noticed that you haven't added yourself to the authors. Would you like to? If so which name/email would you like to use?

Reply:
Thank you for asking! It's fine by me, I'm glad `sock_diag` is officially supported now :tada: